specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/configuration.json

```diff
@@ -0,0 +1,264 @@
+            "$ref": "#/parameters/ConfigurationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The configuration has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryConfiguration"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/configuration": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of configuration at the given location.": {
+            "$ref": "./examples/Configuration/ListConfiguration.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryConfiguration"
+        ],
+        "description": "Returns a list of configuration at the given location.",
+        "operationId": "ContainerRegistryConfiguration_List",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of configuration has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryConfigurationList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerRegistryConfiguration": {
+      "description": "Container registry configuration property.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Container registry configuration property.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ContainerRegistryConfigurationProperty"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "containerregistry.json#/definitions/Resource"
+        }
+      ]
+    },
+    "ContainerRegistryConfigurationProperty": {
+      "description": "Container registry configuration property.",
+      "type": "object",
+      "properties": {
+        "maximumCapacityInGb": {
+          "description": "Total storage capacity (GB) which can used by the registry.",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "ContainerRegistryConfigurationList": {
+      "description": "List of configuration.",
+      "type": "object",
+      "properties": {
```

Comment:
no possibility of paging?

Reply:
no, it is singleton, only 1 item is expected for List